env.dev.example

```diff
@@ -0,0 +1,13 @@
+ADMIN_EMAILS="admin@news.org"
+HOST='localhost:3000'
+PORT=3000
+
+# Database settings
+DATABASE_URL=postgresql://<APP USERNAME>:<APP PASSWORD>@<AURORA WRITER INSTANCE ENDPOINT>:<PORT>/<DB NAME>
```

Comment:
@emamd this is the bit that I found a little mystifying. In our apps, we generally break this into multiple env variables (e.g. `POSTGRES_DB` `POSTGRES_PASSWORD`, etc), but I can't get the app to connect to the remote db this way. It seems that, when set, `DATABASE_URL` supersedes settings in `config/database.yml` (which is where we'd reference all those env variables), but when it's unset and I instead try our typical & more verbose route, it complains about not being able to find the db. 🤷 doesn't seem like a huge deal because it's working, but if you want to get to the bottom of it with me, lmk!